h2o-extensions/target-encoder/src/main/java/ai/h2o/targetencoding/TargetEncoderModel.java

```diff
@@ -84,48 +85,79 @@ protected boolean defaultDropConsCols() {
 
     public final TargetEncoderParameters _parms;
     public final int _nclasses;
+    public final ColumnsToSingleMapping[] _input_to_encoding_column; // maps input columns (or groups of columns) to the single column being effectively encoded (= key in _target_encoding_map).
```

Comment:
Knit: These comments belong above `ColumnsToSingleMapping` and `ColumnsMapping` classes IMHO.

Reply:
Well, those classes can be technically used for any kind of columns to column(s) mapping, it's only those specific instances that tell what we're mapping.